EIPS/eip-7748.md

```diff
@@ -206,7 +205,6 @@ def state_convert(state: State, stride: int):
             
             for chunk_num in range(len(chunked_code)):
                 state_set_codechunk(state, address, chunk_num, chunked_code[chunk_num])
-                n += 1
```

Comment:
See the `Account code chunking done in a single step` rationale section.

Reply:
makes sense lgtm